src/test/java/org/apache/bcel/data/DummySignatureParser.java

```diff
@@ -0,0 +1,1049 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.bcel.data;
+
+/*
+ * The following code is taken from the JDK 8 sources for sun.reflect.generics.
+ * It has been modified to compile stand alone within the BCEL test environment.
+ * The code is not executed but the classfile is used as input to a BCEL test case.
+ */
+
+/*
```

Comment:
-1: We cannot include GPL 2 or GPL 3 code. Please see:
- https://www.apache.org/licenses/GPL-compatibility.html
- https://www.apache.org/foundation/license-faq.html#GPL